apps/web/app/(app)/compose/ComposeEmailForm.tsx

```diff
@@ -166,12 +171,12 @@ export const ComposeEmailForm = (props: {
                 multiple
                 nullable={true}
               >
-                <div className="flex w-full flex-wrap items-center">
+                <div className="flex min-h-10 w-full flex-1 flex-wrap items-center gap-2 rounded-md border border-gray-300 px-2 py-2 shadow-sm focus-within:border-black focus-within:ring-black disabled:cursor-not-allowed disabled:bg-gray-50 disabled:text-gray-500 disabled:ring-gray-200 sm:text-sm">
                   {selectedEmailAddressses.map((emailAddress) => (
                     <Badge
                       key={emailAddress}
                       variant="outline"
-                      className="mr-1.5"
+                      className="h-8 rounded-md border-primary bg-primary text-white"
```

Comment:
mostly not using x-primary but the color instead. sometimes primary appears in codebase but it's rare and i'd prefer to stay consistent with colors here

Reply:
Done!